pkg/api/types.go

```diff
@@ -2262,6 +2262,11 @@ const (
 	// Number of Pods that may be running on this Node: see ResourcePods
 )
 
+const (
+	// Namespace prefix for opaque counted resources (alpha).
+	ResourceOpaqueIntPrefix = "opaque-int.alpha.kubernetes.io/"
```

Comment:
Bikeshed/naming: does anyone prefer the prefix `"alpha.kubernetes.io/opaque-"` to match the existing `""alpha.kubernetes.io/nvidia-gpu"` resource?